.codecov.yml

```diff
@@ -12,6 +12,7 @@ coverage:
         paths:
           - "src/imitation/envs/examples/"
           - "src/imitation/scripts/"
+          - "!src/imitation/scripts/config"
```

Comment:
Why do we need to remove it here? The target is fixed at 0% for auxiliary so it'll never fail the check. Seems useful to still at least calculate the code coverage change.

Reply:
Ah, that's right. I was confusing it with the project auxiliary